src/components/browse/components/FacetList/FacetTermsList.js

```diff
@@ -339,20 +413,49 @@ const ListOfTerms = React.memo(function ListOfTerms(props){
     } = useMemo(function(){
         const { field } = facet;
 
-        const allTermComponents = terms.map(function(term){
+        const facetSearchActive = searchType === 'basic' && searchText && typeof searchText === 'string' && searchText.length > 0;
+        const {
+            filteredTerms: textFilteredTerms = {},
+            filteredSubTerms : textFilteredSubTerms = null
+        } = facetSearchActive ? getFilteredTerms(terms, searchText, facetHasGroupBy) : {};
+
+        const allTermComponents = _.flatten(terms.map(function (term) {
             const { field: currFilteringField, term: currFilteringTerm } = filteringFieldTerm || {};
             const isFiltering = field === currFilteringField && term.key === currFilteringTerm;
-            return <Term {...{ facet, term, termTransformFxn, isFiltering, useRadioIcon }} onClick={onTermClick} key={term.key} status={getTermStatus(term, facet)} />;
-        });
+            const status = getTermStatus(term, facet);
+            const active = status === 'omitted' || status === 'selected';
+            // build tooltip
+            let tooltip = null;
+            if (facetSearchActive && !active && textFilteredTerms[term.key] === true && term.terms && textFilteredSubTerms) {
+                const termName = facet.tooltip_term_substitue || 'term';
+                const filteredTerms = _.filter(term.terms, function (t) { return textFilteredSubTerms[t.key]; });
+                const diff = term.terms.length - filteredTerms.length;
+                tooltip = `Warning: ${term.terms.length} ${termName}${term.terms.length > 1 ? 's' : ''} ${!active ? 'will be' : 'are'} selected`;
+                if (diff > 0) {
+                    if (active) {
+                        tooltip += ` (${diff} currently selected ${termName}${diff > 1 ? 's are' : ' is'} hidden)`;
+                    }
+                    tooltip += `<br />To see all ${facet.tooltip_term_substitue || 'term'}s in this group clear the search filter`;
+                }
+            }
+            if (status !== 'partial') {
+                return <Term {...{ facet, term, termTransformFxn, isFiltering, useRadioIcon, getTermStatus, textFilteredTerms, textFilteredSubTerms, facetSearchActive, tooltip, status }} onClick={onTermClick} key={term.key} />;
+            } else {
+                //duplicate terms to show parent-children tree in active and unselected sections
+                return [
+                    <Term {...{ facet, term, termTransformFxn, isFiltering, useRadioIcon, getTermStatus, textFilteredTerms: {}, status }} onClick={onTermClick} key={term.key} hideUnselectedSubTerms />,
+                    <Term {...{ facet, term, termTransformFxn, isFiltering, useRadioIcon, getTermStatus, textFilteredTerms, textFilteredSubTerms, facetSearchActive, tooltip, status: 'none' }} onClick={onTermClick} key={term.key} hideActiveSubTerms />
```

Comment:
It would make sense to have a `commonProps` object containing `facet`, `term`, `termTransformFxn`, `isFiltering`, `useRadioIcon`, and `getTermStatus`, then pass that (de-structured) into each Term. It'll be a little cleaner/easier to see what's actually different between these various cases here.